src/test/java/com/google/step/TestConstants.java

```diff
@@ -18,17 +20,25 @@
   public static final String BLOBKEY_A = "a_blob_key";
   public static final String BLOBKEY_B = "a_blob_key_b";
 
+  private static final String URL_PREFIX = "/api/images/";
+  public static final String BLOBKEY_URL_A = URL_PREFIX + BLOBKEY_A;
+  public static final String BLOBKEY_URL_B = URL_PREFIX + BLOBKEY_B;
+
   public static final String BIO_A = "Hello world.";
   public static final String BIO_A_NEW = "Hi, I'm Alice";
   public static final String BIO_B = "Hello I'm Bob.";
   public static final String BIO_C = "";
 
+  public static final User USER_A = new User(USER_ID_A, EMAIL_A, USERNAME_A, BLOBKEY_A, BIO_A);
+  public static final User USER_B = new User(USER_ID_B, EMAIL_B, USERNAME_B, BLOBKEY_B, BIO_B);
+  public static final User USER_C = new User(USER_ID_C, EMAIL_C, USERNAME_C, BIO_C);
+
   public static final String USER_A_BRIEF_JSON =
```

Comment:
are the json strings going to be used in more than one file? Because if their only purpose is to test JsonFormatter, then maybe they can be defined inside JsonFormatterTest

Reply:
Some servlet tests also need json string I think

Reply:
ok, then keep it in